examples/shader/Program.cs

```diff
@@ -13,10 +13,7 @@ protected Effect(string name)
             myName = name;
         }
 
-        public string Name
-        {
-            get { return myName; }
-        }
+        public string Name => myName;
```

Comment:
I've made a forum post regarding .Net Core upgrade [here](https://en.sfml-dev.org/forums/index.php?topic=22150.0)
But basicaly if we target .Net Core 1.1 (this PR) the minimum version of .Net Framework needed to use the the binding would be 4.5 or greater